### PR TITLE
Add missing targets

### PR DESCRIPTION
**Pull request by QuentinBisson**
Add missing scrape targets https://github.com/giantswarm/giantswarm/issues/15840

## Checklist

I have:

- [x] Described why this change is being introduced
- [x] Separated out refactoring/reformatting in a dedicated PR
- [x] Updated changelog in `CHANGELOG.md`

### CODE REVIEW

File: files/templates/scrapeconfigs/additional-scrape-configs.template.yaml [hunk 1]
```diff
@@ -1684,6 +1684,131 @@
...
+  params:
+    module: [upstream_check]
+  static_configs:
+  - targets:
+    - 10.127.3.131:30038
+    - 10.127.3.132:30038
+    - 10.127.3.133:30038
+    - 10.127.3.134:30038
+    - 10.127.3.135:30038
+    - 10.127.3.136:30038
```
Comment (TheoBrigitte) [inline files/templates/scrapeconfigs/additional-scrape-configs.template.yaml:1715]: I'd appreciate if this list could be sorted.

File: files/templates/scrapeconfigs/additional-scrape-configs.template.yaml [hunk 2]
```diff
@@ -1697,6 +1822,12 @@
     - [[ .Vault ]]:9005
     labels:
       app: cert-exporter
+[[- if or (eq .Installation "dragon") (eq .Installation "dinosaur") ]]
+  - targets: 
+    - [[ .Vault ]]:9101
+    labels:
+      app: haproxy-exporter
+[[- end ]]
```
Comment (TheoBrigitte) [inline files/templates/scrapeconfigs/additional-scrape-configs.template.yaml:1830]: I do hate this condition, do we really need this? If yes please document why and/or link to some issue.

**Review by TheoBrigitte — CHANGES_REQUESTED** [review]

File: files/templates/scrapeconfigs/additional-scrape-configs.template.yaml (hunk 1 above)
Comment (QuentinBisson) [reply files/templates/scrapeconfigs/additional-scrape-configs.template.yaml:1715]: Those targets are all failing on puma and the blackbox exporter is no more so I will remove them

Comment (QuentinBisson) [reply files/templates/scrapeconfigs/additional-scrape-configs.template.yaml]: Sorted

**Review by mig4 — APPROVED** [review]
LGTMIIWFY

**Pull request merged**